gensrc/proto/internal_service.proto

```diff
@@ -63,6 +63,9 @@ message PTransmitDataParams {
     optional PQueryStatistics query_statistics = 8;
 };
 
+message PHttpRequest {
```

Comment:
What's the usage of this empty protobuf message?

Reply:
just a placeholder but necessary.